templates/default/rt/rtqueueedit.html

```diff
@@ -247,9 +247,9 @@ <H1>{$layout.pagetitle}</H1>
 	</TR>
 	<TR>
 		<TD class="text-right">
-			<A HREF="javascript:document.queue.submit();" accesskey="s"><IMG src="img/save.gif" alt=""> {trans("Submit")}</A>
-			<A id="delete-rtqueue" HREF="?m=rtqueuedel&id={$queue.id}&qaction=delete" accesskey="d" data-name="{$queue.name}"><IMG src="img/delete.gif" alt=""> {trans("Delete")}</A>
-			<A HREF="?m=rtqueuelist"><IMG src="img/cancel.gif" alt=""> {trans("Cancel")}</A>
+			{button type="submit" icon="save" label="Submit"}
+			{button id="delete-rtqueue" href='location.href="?m=rtqueuedel&id={$queue.id}&qaction=delete"' accesskey="d" data-name="{$queue.name}" icon="delete" label="Delete"}
```

Comment:
href dla buttona raczej nie zadziała. Pewnie więcej takich kwiatków.
Możliwe, że warto dla buttona normalnego typu zrobić od razu obsługę parametru `href` tak, żeby generowany html miał `onclick="location.href='....';"`

Reply:
To nie jest głupi pomysł. Na tą chwilę zamieniłem href na onclick.